compiler/rustc_codegen_llvm/src/coverageinfo/mapgen.rs

```diff
@@ -331,13 +334,35 @@ fn generate_coverage_map<'ll>(
     );
 
     // Create the complete LLVM coverage data value to add to the LLVM IR
-    cx.const_struct(&[cov_data_header_val, filenames_val], /*packed=*/ false)
+    let covmap_data =
+        cx.const_struct(&[cov_data_header_val, filenames_val], /*packed=*/ false);
```

Comment:
Discussion (not for this PR): `enum Packed { Yes, No }` over plain bool, considering there's `/*packed=*/` inline comments *anyway*.